### ISSUE TITLE
Crash report 8.1: RuntimeException in AztecParser.markBlockElementsAsParagraphs

### ISSUE THREAD

**Issue by khaykov** [wordpress-mobile/AztecEditor-Android#481]
So far this issue only reported on Android 7.0 and 4.1.2 on following **Samsung** devices:

- Galaxy S7 Edge
- Galaxy S7
- Galaxy Win

```
Fatal Exception: java.lang.RuntimeException: Unable to start activity ComponentInfo{org.wordpress.android/org.wordpress.android.ui.posts.EditPostActivity}: java.lang.RuntimeException: PARAGRAPH span must start at paragraph boundary
       at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:2114)
       at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:2139)
       at android.app.ActivityThread.access$700(ActivityThread.java:143)
       at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1241)
       at android.os.Handler.dispatchMessage(Handler.java:99)
       at android.os.Looper.loop(Looper.java:137)
       at android.app.ActivityThread.main(ActivityThread.java:4960)
       at java.lang.reflect.Method.invokeNative(Method.java)
       at java.lang.reflect.Method.invoke(Method.java:511)
       at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:1038)
       at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:805)
       at dalvik.system.NativeStart.main(NativeStart.java)
Caused by java.lang.RuntimeException: PARAGRAPH span must start at paragraph boundary
       at android.text.SpannableStringBuilder.setSpan(SpannableStringBuilder.java:600)
       at android.text.SpannableStringBuilder.setSpan(SpannableStringBuilder.java:588)
       at org.wordpress.aztec.util.SpanWrapper.setFlags(SpanWrapper.kt:20)
       at org.wordpress.aztec.AztecParser.markBlockElementsAsParagraphs(AztecParser.kt:255)
       at org.wordpress.aztec.AztecParser.fromHtml(AztecParser.kt:45)
       at org.wordpress.android.editor.AztecEditorFragment.hasMediaItemsMarkedWithTag(AztecEditorFragment.java:1623)
       at org.wordpress.android.editor.AztecEditorFragment.hasMediaItemsMarkedUploading(AztecEditorFragment.java:1613)
       at org.wordpress.android.ui.posts.EditPostActivity.resetUploadingMediaToFailedIfPostHasNotMediaInProgressOrQueued(EditPostActivity.java:421)
       at org.wordpress.android.ui.posts.EditPostActivity.onCreate(EditPostActivity.java:363)
       at android.app.Activity.performCreate(Activity.java:5203)
       at android.app.Instrumentation.callActivityOnCreate(Instrumentation.java:1094)
       at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:2078)
       at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:2139)
       at android.app.ActivityThread.access$700(ActivityThread.java:143)
       at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1241)
       at android.os.Handler.dispatchMessage(Handler.java:99)
       at android.os.Looper.loop(Looper.java:137)
       at android.app.ActivityThread.main(ActivityThread.java:4960)
       at java.lang.reflect.Method.invokeNative(Method.java)
       at java.lang.reflect.Method.invoke(Method.java:511)
       at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:1038)
       at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:805)
       at dalvik.system.NativeStart.main(NativeStart.java)
```

**Comment by daniloercoli** [comment]
See https://fabric.io/automattic/android/apps/org.wordpress.android/issues/59a7e418be077a4dcce162ef?time=last-thirty-days

**Comment by daniloercoli** [comment]
Fixed in #539
